documentation/writing-providers.md

```diff
@@ -293,13 +308,25 @@ In the repo root, open `.goreleaser.yml` and add the provider to `Provider-speci
 
 Here are some last-minute things to check before you submit your PR.
 
-1. Run `go generate` to make sure all generated files are fresh.
-2. Make sure all appropriate documentation is current. (See [Step 8](#step-8-manual-tests))
-3. Check that dependencies are current (See [Step 13](#step-13-dependencies))
-4. Re-run the integration test one last time (See [Step 7](#step-7-integration-test))
+1. Run `go generate ./...` to make sure all generated files are fresh.
+2. Make sure the following files were created and/or updated:
+  * `OWNERS`
+  * `README.md`
+  * `.github/workflows/pr_test.yml` (The PROVIDERS list)
+  * `.goreleaser.yml` (Search for "Provider-specific changes")
+  * `documentation/SUMMARY.md`
+  * `documentation/providers.md` (the autogenerated table + the second one; make sure it is removed from the "requested" list)
+  * `documentation/providers/`PROVIDERNAME`.md`
+  * `integrationTest/providers.json`
+  * `providers/_all/all.go`
+3. Verify you're using the most recent version of anything you import.  (See [Step 13](#step-13-dependencies))
+4. Re-run the integration test one last time.
+  * (See [Step 7](#step-7-integration-test)).
+  * Post the results as a comment to your PR.
 5. Re-read the [maintainer's responsibilities](providers.md) bullet list.  By submitting a provider you agree to maintain it, respond to bugs, periodically re-run the integration test to verify nothing has broken, and if we don't hear from you for 2 months we may disable the provider.
 
 ## Step 16: After the PR is merged
 
 1. Remove the "provider-request" label from the PR.
 2. Verify that [documentation/providers.md](providers.md) no longer shows the provider as "requested"
```

Comment:
"Step 16: After the PR is merged" is no longer the current method. This changed with the migration to GitBook.

Reply:
This GitHub thread can be marked as resolved. 👍